Research/julia/src/README.md

```diff
@@ -0,0 +1,96 @@
+---
+page_type: sample
+languages:
+  - csharp
+products:
+  - azure
+  - azure-media-services
+  - azure-live-video-analytics
+description: "The sample in this repo shows how to use the Live Video Analytics on IoT Edge module to analyze live video using AI modules of your choice and record video on the Azure Stack Edge device."  
+---
+
+This tutorial will walk you through setting up an Azure Stack Edge device and deploying the following modules:
+- Live Video Analytics
+- RTSP simulation
+- Yolo V3 machine learning model to detect objects
+
+You can choose to set this up so the RTSP simulation videos are uploaded to the cloud or used locally. This tutorial will set things up to run locally.
+
+# Live Video Analytics on Azure Stack Edge samples
+
+This repository contains C# samples for Live Video Analytics on Azure Stack Edge
+
+## Contents
+
+| File/folder       | Description                                |
+|----------------------|--------------------------------------------|
+| `src`                | Sample source code.                        |
+| `.gitignore`         | Defines what to ignore at commit time.     |
+| `README.md`          | This README file.                          |
+| `LICENSE`            | The license for the sample.                |
+| `SECURITY`           | Guidelines for reporting security issues   |
+| `CODE_OF_CONDUCT.md` | Open source code of conduct                |
```

Comment:
I don't think you do or should have these files. I'm guessing you copy/pasted from somewhere? Remove, and check if you missed any other cleanup in this file.